Makefile

```diff
@@ -17,62 +17,65 @@ BUILDFLAGS ?=
 
 ifdef GOBIN
 PATH := $(GOBIN):$(PATH)
-else ifdef GOPATH
-PATH := $(subst :,/bin:,$(GOPATH))/bin:$(PATH)
 else
-PATH := $(HOME)/go/bin:$(PATH)
+PATH := $(subst :,/bin:,$(shell go env GOPATH))/bin:$(PATH)
 endif
 
 LDFLAGS := $(LDFLAGS) -X main.commit=$(COMMIT) -X main.branch=$(BRANCH)
 ifdef VERSION
 	LDFLAGS += -X main.version=$(VERSION)
 endif
 
+.PHONY: all
 all:
-	$(MAKE) deps
-	$(MAKE) telegraf
+	@$(MAKE) --no-print-directory deps
+	@$(MAKE) --no-print-directory telegraf
 
+.PHONY: ensure
```

Comment:
did you mean `deps` instead of `ensure`?

Reply:
ensurely I did